### PR TITLE
versioning: fix outstanding issues

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
Fixes versioning so it can be merged
Notes:
- Unifies saga editing callsites
- Does not depend on kernel kicking the subscription so works correctly with fake marks

### CODE REVIEW

**Comment by midsum-salrux** [comment]
```
gall: poke-as cast fail :groups [a=%json b=%group-create]
[%key 'secret']
/sys/vane/clay/hoon:<[832 51].[832 63]>
/sys/vane/gall/hoon:<[1.058 27].[1.058 45]>
```

getting this error in the JS console when I try to create a group to test this with

**Comment by arthyn** [comment]
@midsum-salrux you'll need to run the UI preview, if you check out DEVELOPMENT.md it has instructions under getting started

File: desk/lib/wood.hoon [hunk 1]
```diff
@@ -0,0 +1,31 @@
+::  $wood: logging
```
Comment (ryjm) [inline desk/lib/wood.hoon:1]: nice, thoughts on adding levels / `%stir`-like poke too?

**Review by ryjm — COMMENTED** [review]
looks good (logging lib yes plz) but why remove keep?

**Comment by jamesacklin** [comment]
@midsum-salrux Can you reproduce this kick-loop in this branch? https://github.com/tloncorp/landscape-apps/pull/1031#issuecomment-1322263026

**Comment by midsum-salrux** [comment]
> @midsum-salrux Can you reproduce this kick-loop in this branch? [#1031 (comment)](https://github.com/tloncorp/landscape-apps/pull/1031#issuecomment-1322263026)

Was testing with a pair of livenet moons that I got into a bad state. Going to try to get a more reproducible fakezod setup.

**Comment by liam-fitzgerald** [comment]
yes @ryjm definitely should flesh out the logging lib a bit more, just
wanted to get something going

**Review by arthyn — APPROVED** [review]
As far as I can tell this looks good, chi/dex/lev seems consistent across the agents.

**Pull request merged**